### ISSUE TITLE
Deploy on netlify

### ISSUE THREAD

**Issue by jayvdb** [gsocindonesia/gsocindonesia.github.io#3]
http://netlify.com/ is a new service provider with lots of cool features, including forms, webhooks, and deploy previews.

The site can be hosted on both GitHub Pages and Netlify at the same time.

https://github.com/jayvdb/jekyll-netlify is a nice addition that myself and @bekicot have been working on to add extra features to make the integration tighter.

**Comment by ceefour** [comment]
@jayvdb Thanks for the tip.

My opinion is at this point it is more important to have a useful project for students to work/contribute on to get experience in safe and easy way. Your GSoC browser is a cool idea and I totally dig it.

@bekicot's https://bekicot.github.io/gsoc-prep-tasks/ is already a very good start and he can either expand/improve on that (i.e. making one project larger and more complex) or create a new project (small, focused projects). I think small miniprojects will have less barrier of entry for beginners. Other students can contribute in various little steps, like:

* documentation
* translation
* unit tests
* prettifying the layout
* adding graphics
* adding animations
* data analysis (actually the organization selection part, it's so much more about data analysis then coding!)
* etc. 

Doesn't have to involve coding (at start). If @bekicot is interested in this, please do. :)

However contributing gsocindonesia.github.io "website" itself would be a good intro for beginners. They can fix/clarify existing pages, add pages, etc. overall more "writing" work than "coding" work, but goal is to be familiar with how GitHub and collaboration works, and a bit of Markdown "tech".

As for the current state: simply filling the Google Form and creating a GitHub Issue is already an accomplishment, by overall standards. :) I'm hoping next year the bar can be set higher, but we have to start from somewhere. :)

**Comment by ceefour** [comment]
BTW I don't disagree about the netlify thing.

What I meant is that it should be a student contribution. i.e. if a student can be assigned to this issue, then start writing a summary, analysis of netlify features, even just as issue comments here (even in Indonesian), that'd be great. Still counts as green dots in profile right? And is pretty meaningful. Much better if can do like breakdown of website structure, design of form, how to handle form submission. Just starting would already be great. :)

**Comment by ceefour** [comment]
I think I'll start a page of "Ideas List" for @gsocindonesia itself. Kind of GCI tasks, categorized by difficulty level:

* newcomer=translation, fixing README, fixing typo, adding graphics; 
* basic=fixing CSS, fixing layout, drawing pretty techy diagrams, flowcharts, jquery animations; 
* intermediate="coding", data structures, algorithm analysis). 

That participants or would-be participants (not yet registered, still not convinced/interested/PeDe) can take a look and then think.. "hey this isn't as bad as I thought". Most tasks would be in Indonesian as well, just to be less intimidated to "dip a toe in water".

It's possible this is a redundant effort and perhaps you can point me to already existing list. 

Problem is, the majority of students has never even heard of GitHub, and even those who have a GitHub account, don't actively use it:

![image](https://user-images.githubusercontent.com/24123/36638635-0ef6543e-1a2d-11e8-9280-c32ace4b1b08.png)

So part of the plan is I guess just to get them to use GitHub in a meaningful way and see how it works, regardless of their "coding" skill.

**Comment by jayvdb** [comment]
Yup, I am just creating issues for useful things that motivated students can do.

However in some cases, using better tools will mean that a higher investment initially will pay off quickly with a time saving in the short term for your coach assistants, such as https://github.com/gsocindonesia/gsocindonesia.github.io/issues/5 .

Created https://github.com/gsocindonesia/gsocindonesia.github.io/issues/6 for the GSoC org listing idea.

**Comment by ceefour** [comment]
Note to self: It's unfortunate that I didn't ask students:

* What level is their usage of GitHub? Didn't know / Why should I use GitHub? / Interested to create account / Have account but not used / Sometimes I use GitHub / I use GitHub
* What level is their English, reading vs writing? I can't read English / I can use English but I prefer Indonesian / I can read English but bad at writing / I can write English but bad at reading / English is good

**Comment by ceefour** [comment]
@jayvdb "using better tools will mean that a higher investment initially will pay off quickly", agree, though someone has to do and it'd be great if that someone(s) are students :) If they fail etc. it won't hinder the program's own progress, and it's still achievement on their part (learn to contribute, etc.)

**Comment by ceefour** [comment]
So if anyone wishes to take this on (while still completing their GsoC proposals!), it'd be really great. Could be @wisn @bekicot @probeadd @afnizarnur @afnizarnur or any other student really. And this isn't a "big complex" thing, it's a fun to do in leisure, and it'd be even better if you can divide the issues into smaller tasks that you can the distribute to other fellow students :) Oh and it doesn't have to be in English (John can use Google Translate if needed :p )

**Comment by wisn** [comment]
I would love to use static site first then improve it using static page generator such as Jekyll. @afnizarnur could provide the design template so I can start to code. Otherwise, I will just code using my average sense of art.

**Comment by wisn** [comment]
> even just as issue comments here (even in Indonesian), that'd be great. Still counts as green dots in profile right?

Give a comment on an issue won't count as a contribution.

**Comment by ceefour** [comment]
@wisn I see, I guess start from issue comments (to make familiar with GitHub) then move as quickly as possible to light contributions like copy-pasting that issue comments into `my-task-summary.md`  thing :)

**Comment by ceefour** [comment]
I have a theory, students are afraid of GitHub because they don't know why it's beneficial, and second because it's "a coding thing".

So the sooner they can accept that GitHub as "just like another Facebook/Instagram" where people can comment on each other's ideas, the better. The issue (pun intended) is that in GitHub-speak, a "status update" is called an Issue 😅

**Comment by wisn** [comment]
Ah, you are right. GitHub is just another social media 🤣
